--- blockdiag-1.1.1/src/blockdiag/utils/fontmap.py	2011-12-04 21:55:40.867031987 +0200
+++ blockdiag-1.1.1/src/blockdiag/utils/fontmap.py	2011-12-04 21:55:40.867031987 +0200
@@ -18,7 +18,12 @@
 import sys
 import copy
 import codecs
-from ordereddict import OrderedDict
+try:
+    # py2.7
+    from collections import OrderedDict
+except ImportError:
+    # python-ordereddict
+    from ordereddict import OrderedDict
 from ConfigParser import SafeConfigParser
 from blockdiag.utils.collections import namedtuple
 
@@ -93,7 +98,7 @@
         self.set_default_font(None)
 
     def _parse_config(self, conffile):
-        if sys.version_info > (2, 6):
+        if sys.version_info > (2, 4):
             config = SafeConfigParser(dict_type=OrderedDict)
         else:
             config = SafeConfigParser()
